Common/src/main/java/com/tabnineCommon/capabilities/CapabilitiesService.java

```diff
@@ -30,9 +30,7 @@ public static CapabilitiesService getInstance() {
   }
 
   public void init() {
-    if (!Config.IS_SELF_HOSTED) {
-      scheduleFetchCapabilitiesTask();
-    }
+    scheduleFetchCapabilitiesTask();
```

Comment:
why?

Reply:
it won't be called anyway when self-hosted.